Month3/LinkedList/LinkedList/TSLList.cpp

```diff
@@ -0,0 +1,178 @@
+#define _CRTDBG_MAP_ALLOC
+#include <stdlib.h>
+#include <crtdbg.h>
+#include <iostream>
+#include <string>
+#include"TSLList.h"
+
+template<typename T>
+void PrintList(TSLList<T>& List)
+{
+	std::cout << "Size: " << List.GetSize() << "\t\tList Contents: { ";
+	for (const T& Element : List)
+	{
+		std::cout << Element << ' ';
+	}
+	std::cout << "}\n";
+}
+
+template<typename T>
+void PrintHead(const TSLList<T>& List, const std::string& CorrectHead)
+{
+	std::cout << "Head is:\t" << List.GetHead() << (List.GetHead() == CorrectHead ? " CORRECT" : " INCORRECT") << std::endl;
+}
+
+template<typename T>
+void PrintTail(const TSLList<T>& List, const std::string& CorrectTail)
+{
+	std::cout << "Tail is:\t" << List.GetTail() << (List.GetTail() == CorrectTail ? " CORRECT" : " INCORRECT") << std::endl;
+}
+
+int main()
```

Comment:
this file should be called main or source, anything but TSLList is confusing

Reply:
itself is the application of using SLList, so in that context it should not be confusing